tests/components/eheimdigital/snapshots/test_diagnostics.ambr

```diff
@@ -0,0 +1,26 @@
+# serializer version: 1
+# name: test_entry_diagnostics
+  dict({
+    'data': None,
```

Comment:
Can we get a proper data entry here?

Reply:
Hmm, should I use a fixture json for this? The mocks will not work for this, I think.

Reply:
Okay, I think I need to learn how to use the mocks correctly, currently I mock the properties of the objects, as from testing it doesn't work when I set the data object as calling properties doesn't seem to work at all.

Reply:
Can you push what you have?

Reply:
I have it pushed, my problem is that I don't know how to have the mock call the property functions, so that they look up the data in the dicts.

Reply:
you don't because your mock isn't tied to your object anymore

Reply:
Okay, I will revamp the test structure, I have an idea about what to do, but I will do it in a separate PR.

Reply:
I think just adding the properties that are read is good enough though!

Reply:
I did the fixture json approach as this is better than mocking the properties and I don't have to have the data there twice just to satisfy the diagnostics.